### DISCUSSION TITLE
Including this video to the front page!

### DISCUSSION THREAD

**Discussion by danielgallab** [FlagClicked/FlagClicked#11] [Ideas]
Made by Ryker J... this is epic :)

https://user-images.githubusercontent.com/72760579/111247846-20d58f00-85df-11eb-8a92-123474cd1cca.mp4

I'm thinking about having this for first-time viewers of the site... Looks sick and people will love it :)

**Comment by retronbv** [comment]
how will we track first time viewers?

**Reply by danielgallab** [reply]
document.cookie

**Reply by retronbv** [reply]
ok

**Reply by markverb1** [reply]
[No one has ever done this before, yeah yeah totally! /s](https://dbd.js.org)

**Reply by markverb1** [reply]
> document.cookie

Or local storage

**Reply by markverb1** [reply]
> > > document.cookie
> > 
> > 
> > Or local storage
> 
> @SheepTester when you open a page using local storage for the first time: "I don't care about your privacy."

*cookies

**Reply by SheepTester** [reply]
That warning shows up on all pages regardless of whether it uses localStorage if I made it before 2018 or 2019 I think. Also, my website doesn't use cookies

I believe the EU cookie law applies to localStorage (I don't think the law itself explicitly says cookies; "cookie law" is an informal name), but it might not apply if you just use it to store something in the browser only, without sending it to the backend. Not sure, not a lawyer

**Comment by Steve0Greatness** [comment]
lol, I'm trying to learn how to use cookies rn... super confused.

**Reply by SheepTester** [reply]
For cases like these where it can just be for the browser, you should use localStorage rather than cookies.

In JavaScript on the web page, `localStorage` is provided and acts like an object that is saved between page reloads (though it only can store strings). So you can do

```js
// localStorage.seenVideo will be undefined if it hasn't been set
if (!localStorage.seenVideo) {
  showVideo()
  localStorage.seenVideo = 'true'
}
```

**Reply by Steve0Greatness** [reply]
I've tried...
![Screenshot 2021-05-23 9 57 56 AM](https://user-images.githubusercontent.com/75220768/119269594-5da89e00-bbad-11eb-9962-7ba50ea3b4d1.png)

**Reply by SheepTester** [reply]
I'm not super familiar with Vue, but perhaps you meant to `return showVideo()`?

**Reply by Steve0Greatness** [reply]
> I'm not super familiar with Vue, but perhaps you meant to `return showVideo()`?

I'm not sure... where should it go?

**Comment by PoIygon** [comment]
What video editor did you make this on

**Reply by Steve0Greatness** [reply]
You'll have to ask @TheRykerJ
